hw/ip/pwm/dv/cov/manual_excl.vRefine

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<refinement-file-root>
+  <rules>
+    <!--
+        Waive block coverage for a conditional which is true if the 'we' signal is true for the
+        REGWEN in register in prim_subreg_arb. If the signal is true, this denotes a write to the
+        register from hardware but the instantiation in pwm_reg_top wires it to zero.
```

Comment:
Sorry Rupert, I don't understand this: in pwm_reg_top.sv the `we` input of `u_regwen` is connected to `regwen_we`, and from this comment I thought that is what you refer to. Could you clarify?

Reply:
I was busy writing a careful reply until I convinced myself that I'd got this wrong! What you write is completely correct. My suspicion is that the coverage hole was because the tests weren't actually writing to REGWEN at all.

Sorry for the noise!